subgraphs/aave-v2/src/mappings/utilFunctions.ts

```diff
@@ -0,0 +1,559 @@
+      market.stableBorrowRate = bigIntToBigDecimal(rayToWad(tryReserve.value.currentStableBorrowRate));
+      market.variableBorrowRate = bigIntToBigDecimal(rayToWad(tryReserve.value.currentVariableBorrowRate));
+    } else {
+      log.error('FAILED TO GET RESERVE', []);
+    }
+  }
+  // No need to execute the below code until block 12317479 when incentive controller was deployed and started calculating rewards
+  if (blockNumber.gt(BigInt.fromString('12317479'))) {
+    let rewardTokens = market.rewardTokens;
+    if (rewardTokens === null) rewardTokens = [];
+    if (rewardTokens.length === 0) {
+      // If the reward tokens have not been initialized on the market, attempt to pull them from the incentive controller
+      const rewardTokenFromIncController = Address.fromString(REWARD_TOKEN_ADDRESS);
+      const depositRewardToken = loadRewardToken(rewardTokenFromIncController, RewardTokenType.DEPOSIT);
+      const borrowRewardToken = loadRewardToken(rewardTokenFromIncController, RewardTokenType.BORROW);
+      market.rewardTokens = [depositRewardToken.id, borrowRewardToken.id];
+      market.rewardTokenEmissionsAmount = [BIGINT_ZERO, BIGINT_ZERO];
+      market.rewardTokenEmissionsUSD = [BIGDECIMAL_ZERO, BIGDECIMAL_ZERO];
+    }
+    // Get reward emissions
+    market.rewardTokenEmissionsAmount = getCurrentRewardEmissions(market);
+    market.rewardTokenEmissionsUSD = getCurrentRewardEmissionsUSD(market);
+  }
+  const currentPriceUSD = getAssetPriceInUSDC(Address.fromString(market.id));
+  market.outputTokenPriceUSD = currentPriceUSD;
+  market.inputTokenPricesUSD = [currentPriceUSD];
+  market.save();
+  return market as Market;
+}
+
+export function getCurrentRewardEmissions(market: Market): BigInt[] {
+  const rewardEmissions = market.rewardTokenEmissionsAmount;
+  // Attempt to get the incentives controller contract
+  const incentivesController = initIncentivesController(market);
+  // From the incentives controller contract, get pull the 'assets' values from the market aToken, sToken, and vToken
+  const incentivesControllerContract = IncentivesControllerContract.bind(incentivesController);
+  if (!incentivesControllerContract.try_assets(Address.fromString(market.outputToken)).reverted) {
+    log.info('REWARD EMISSIONS ' + market.outputToken + ' ' + incentivesControllerContract.assets(Address.fromString(market.outputToken)).value0.toString(), []);
+    const assetDataSupply = incentivesControllerContract.try_assets(Address.fromString(market.outputToken)).value.value0;
+    const assetDataBorrowStable = incentivesControllerContract.try_assets(Address.fromString(market.sToken)).value.value0;
+    const assetDataBorrowVariable = incentivesControllerContract.try_assets(Address.fromString(market.vToken)).value.value0;
+    // Get the emissions per day for the aToken rewards for deposits
+    if (!assetDataSupply.equals(BIGINT_ZERO)) {
+      rewardEmissions[0] = emissionsPerDay(assetDataSupply);
```

Comment:
if it's consistent the 0 is deposit reward and 1 is borrow reward, you should change them into named constants and use throughout